tests/expressions/tests.py

```diff
@@ -2849,3 +2849,34 @@ def test_nulls_false(self):
             F("field").asc(nulls_first=False)
         with self.assertRaisesMessage(ValueError, msg):
             F("field").desc(nulls_last=False)
+
+
+def test_order_by_with_outerref_comprehensive(self):
+    # Test simple subtraction using OuterRef in order_by.
```

Comment:
We avoid docstrings that begin "test ...", see style [guide](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/) on "preambles". Also, if you do need to provide extra context in a comment, do it in a docstring. Again below.